google/cloud/containeranalysis/v1/.repo-metadata.json

```diff
@@ -0,0 +1,14 @@
+{
+    "api_id": "containeranalysis.googleapis.com",
+    "api_shortname": "containeranalysis",
+    "client_documentation": "https://cloud.google.com/cpp/reference/containeranalysis/latest",
+    "distribution_name": "google-cloud-cpp",
+    "issue_tracker": "https://issuetracker.google.com/issues?q=componentid:187156",
```

Comment:
comment: I guess this component is shared by `artifactregistry` :shrug:

Reply:
yes, they do.